app/javascript/crayons/MarkdownToolbar/MarkdownToolbar.jsx

```diff
@@ -8,6 +9,25 @@ import { Overflow, Help } from './icons';
 import { Button } from '@crayons';
 import { KeyboardShortcuts } from '@components/useKeyboardShortcuts';
 import { BREAKPOINTS, useMediaQuery } from '@components/useMediaQuery';
+import { getSelectionData } from '@utilities/textAreaUtils';
+
+const getNextMatchingSibling = (element, selector) => {
```

Comment:
Previously we managed the roving tab index by going directly to next or previous element siblings. Now we introduce the image uploader we can't do that, and have to instead check for a selector match (because there is an additional - hidden - file input in the ImageUploader that we don't want to focus)

Reply:
Yes, good point, I'll add that in